mlir/test/Target/LLVMIR/openmp-target-nesting-in-host-ops.mlir

```diff
@@ -0,0 +1,136 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
```

Comment:
There are more cases to cover and test (e.g. reductions). I will work on that while the PR is reviewed.